asreview/webapp/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://localhost:5000/
```

Comment:
Do you want to commit this (as you seem to ignore it above)?

Reply:
I think I understand after carefully reading your post.

Reply:
Yeah, this is the default. But somehow my port 5000 has been taken, so the app takes 5001. To overcome this I create a .env.local file which overrides .env with the correct port.